lib/Doctrine/ORM/Tools/Pagination/Paginator.php

```diff
@@ -107,6 +107,16 @@ public function count()
 
         return $this->count;
     }
+    
+    /**
+     * Returns last page number
+     *
+     * @return int
+     */
+    public function lastPage()
```

Comment:
getLastPageNumber would not be more appropriate?

Reply:
Or `getPagesCount()`.
Also needs a return type.